app/controllers/concerns/authenticated_state_file_intake_concern.rb

```diff
@@ -12,7 +13,7 @@ def card_postscript
   end
 
   def require_state_file_intake_login
-    if current_state_file_az_intake.blank? && current_state_file_ny_intake.blank?
+    if current_intake.blank?
```

Comment:
Good job - this will subtly change the behavior - previously we would not redirect to login in Ny if there was an Az intake present, and vice versa. Now we will